.github/workflows/build-opencv-base-container.yml

```diff
@@ -10,24 +10,92 @@ on:
     - build/intermediate-containers.mk
     - Makefile
   pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
     branches: [ main ]
     paths:
     - .github/actions/build-intermediate/**
     - .github/workflows/build-opencv-base-container.yml
     - build/containers/intermediate/Dockerfile.opencvsharp-build
     - build/intermediate-containers.mk
     - Makefile
+  issue_comment:
+    types: [created, edited]
+    branches: [ main ]
 
 env:
   AKRI_COMPONENT: opencvsharp-build
   MAKEFILE_COMPONENT: opencv-base
 
 jobs:
 
+  add-same-version-label-to-pr:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issue_comment' && contains(github.event.comment.body, '/add-same-version-label')
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add same version label
+      uses: actions/github-script@0.9.0
+      if: success()
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          github.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ['same version']
+          })
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: '👋 Added [same version] label :)!'
+          })
+
+  add-build-label-to-pr:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issue_comment' && contains(github.event.comment.body, '/add-build-dependency-containers-label')
+    steps:
+    - uses: actions/checkout@v2
+    - name: Add build dependency containers label
+      uses: actions/github-script@0.9.0
+      if: success()
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          github.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ['build dependency containers']
+          })
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: '👋 Added [build dependency containers] label :)!'
+          })
+
+  add-label-missing-comment:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'unlabeled') && contains(github.event.pull_request.labels.*.name, 'build dependency containers') == false

```

Comment:
What is this `github.event.action == 'unlabeled'` trying to capture? What if there is some other label like `bug` or `documentation` on the PR? Seems like just the `opened` expression is sufficient.